Specification/Studies/README.md

```diff
@@ -920,6 +920,44 @@ Linked data
         ],
         "documentationURL": "https://brapi.org",
         "endDate": "2014-01-01",
+        "environmentParameters": [
+            {
+                "parameterName": "soil type",
+                "parameterPUI": "http://purl.obolibrary.org/obo/EO_0007155",
```

Comment:
@PapoutsoglouE can you change EO by PECO please. EO does not exist anymore and has been replaced by PECO. Thanks

Reply:
and for clay soil, ENVO:00002262 should be used instead of PECO. PECO is really for exposure whereas ENVO is more for natural environments

Reply:
Done, thank you!